docroot/themes/custom/uids_base/scss/components/caption.scss

```diff
@@ -1,6 +1,13 @@
 @import "assets/scss/_variables.scss";
 @import "assets/scss/_utilities.scss";
 
+.banner:hover {
```

Comment:
Can we comment this out until we have done further work on this?

Reply:
can do!